core/primitives/src/types.rs

```diff
@@ -1120,7 +1120,7 @@ pub enum ValidatorInfoIdentifier {
     ProtocolSchema,
 )]
 pub enum ValidatorKickoutReason {
-    /// Slashed validators are kicked out.
+    /// Deprecated
     Slashed,
```

Comment:
Prefix `_Unused` like other enums where we want to keep borsh serializability?

Reply:
Good idea.  Fixed.